tests/libvmi/network.go

```diff
@@ -48,3 +52,29 @@ func InterfaceDeviceWithMasqueradeBinding() kvirtv1.Interface {
 		},
 	}
 }
+
+// ValidateVMIandPodIPMatch Checks that the vmi pod and vmi scheme have matching Ip/Ips fields for primary interface
+func ValidateVMIandPodIPMatch(vmi *kvirtv1.VirtualMachineInstance, vmiPod *k8sv1.Pod) error {
```

Comment:
I would say that we move this to test/network/asserts.go since it involves VMI and POD in case this is used by more than on test.

Reply:
Edy suggested its to be on libvmi, @EdDev wdyt please?

Reply:
Good point, passing a pod to the libvmi is a bit odd indeed.
"asserts" smells like it includes "Expect", which we do not want to have in helpers. Maybe "validate"?
Not sure about the name, I'll be good with any at this stage, it can be easily changed later.

Reply:
WIP, not ready to review yet

Reply:
Done, in a separate commit that i will rearrange, when i get your blessing, PTAL, thanks